ibis/backends/tests/test_generic.py

```diff
@@ -1190,9 +1190,8 @@ def test_exists(batting, awards_players, method_name):
             marks=[
                 pytest.mark.broken(
```

Comment:
Probably, we should split this test into two: one for literal and one for typeof.